File: examples-sdk/typescript/501-villog-stdout-dev.ts
#!/usr/bin/env tsx
// 501-villog-stdout-dev — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 501-villog-stdout-dev.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/501-villog-stdout-dev.py
#!/usr/bin/env python3
"""501-villog-stdout-dev — Python SDK equivalent
Compile: vil compile --from python --input 501-villog-stdout-dev.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
